shopinvader_product_brand/schemas/brand.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from __future__ import annotations
+
+from odoo.addons.extendable_fastapi import StrictExtendableBaseModel
+
+
+class ShopinvaderBrand(StrictExtendableBaseModel):
+    id: int
+    name: str
+    sequence: int | None = None
+    description: str | None = None
+    short_description: str | None = None
+    meta_description: str | None = None
+    meta_keywords: str | None = None
+    seo_title: str | None = None
+    url_key: str | None = None
```

Comment:
Also add `redirect_url_key`?

Reply:
This field is a Serialized field.
In v14 it was defined on `shopinvader.binding`.
In v16 it is now added on `abstract.url` in this PR: https://github.com/shopinvader/odoo-shopinvader/pull/1399 
Adding it to schema.